backend/alembic/versions/2551e1a3cb06_add_spam_column_to_stories.py

```diff
@@ -20,7 +20,21 @@ def upgrade():
     op.add_column(
         "stories", sa.Column("spam", sa.Integer(), server_default="0")
     )
+    op.add_column(
```

Comment:
@vinovo  Can you verify it?

Reply:
lgtm. i just removed the migration version w/ conflict, @yus252 @TanayKarve can you try the upgrade head command on your end?